### DISCUSSION TITLE
Welcome to TheDiscordSoundboard Discussions!

### DISCUSSION THREAD

**Discussion by Mayerch1** [Mayerch1/SoundBoard#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by r14v8** [comment]
Wow thank you so much for this one!! 

Everything works fine - but not using the bot cause it disconnects after using it one time and then I have to restart the whole programm. It seems just bugged cause I can't hit Play/Pause button in the software either after the bot disconnected.

**Reply by Mayerch1** [reply]
I published a hotfix 2.4.2 which always disconnects after a replay is done. 
This prevents the described bugs.

Automatic join into the channel of your user should be fixed aswell.

I cannot promise to properly re-implement the changes for continuous streaming, aswell as the changes for the youtube library anytime soon.

**Comment by Marzaq** [comment]
Wonderful work. For some reason the bot appears as Defeaned when it enters a voice channel. What can I do to fix this problem?

**Reply by Mayerch1** [reply]
That is intentend behaviour. 
The bot doesn't need to listen to your audio and it saves a bit of bandwith that way, as the discord gateway doesn't need to deliver the audio data to the bot.

It does not take the ability of the bot to transmit audio
